### ISSUE TITLE
DetailsList not able to navigate with ctrl+alt+ arrowKey in production build

### ISSUE THREAD

**Issue by bensonz** [microsoft/fluentui#19676]
### Environment Information

- **Package version(s)**:  "office-ui-fabric-react": "^7.173.1",
- **Browser and OS versions**: CefSharp.Common" Version="91.1.211"

### Describe the issue:

This behavior is very strange. So our team, [SPMT](https://docs.microsoft.com/en-us/sharepointmigration/introducing-the-sharepoint-migration-tool), uses fluent ui. We have been using DetailsList for a couple of places.
And in debug sessions, accessibility bugs are fine, the detailsList can be navigated through Ctrl+Alt+Arrow Key.
Here we  are using NVDA, I haven't checked Narrator.

We use webpack to build this. Config is really simple:
```
if (env && env.dev === true) {
        console.log("Gen config for development");
        config.mode = 'development';
        config.devtool = 'inline-source-map';
    } else {
        console.log("Gen config for production");
        config.mode = 'production';
    }
```

Here the ctrl+alt+arrow key is working if I just change this
`config.mode = 'production'` to `'development'` 

We have no idea why this happening, but that's what we observed.

### Please provide a reproduction of the issue in a codepen:

This is hard to do, because there's no prod build in codepen.

#### Actual behavior:

Unable to navigate.

#### Expected behavior:

Act as development.

**Comment by bensonz** [comment]
Hello team, any updates?

**Comment by bensonz** [comment]
@GeoffCoxMSFT Hello, can anyone in the accessibility team take a look?

**Comment by GeoffCoxMSFT** [comment]
@bensonz I'll try to reproduce this today in a local sample app.

**Comment by bensonz** [comment]
@GeoffCoxMSFT 
Hmmm it does work...
I have created a sample app at https://github.com/bensonz/purelyTest/tree/main/test-detailslist
Both `start` and `build` work.
I tested it on my own PC and it worked, in fact, it worked on both website and SPMT (out tool). 

I have yet to test on my office's computer, which was failing before. I am a little bit confused about what might have caused this issue now. I did test with SPMT and verified that prod doesnt work while dev works. 

Once I do more testing, I'll update you about this. Closing it for now.
